config/defaults.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     publishMetaData:      'false',
     statsComponent:       undefined,
     awsKeyPrefix:         undefined,
-    iterationInterval:    1000 * 10,
+    iterationInterval:    1000 * 1,
```

Comment:
Hmm, yes, we could do that.  We also have a hard limit of 10 minutes before we process.exit() and I'd like to let the EC2 api cool down between iterations.  If we have a >75s iteration, we probably just melted the AWS api down, and we should take a break of 75s anyway.